Main.java

```diff
@@ -15,7 +17,59 @@ public static void main(String[] args) {
 
    // calculate and print days the debtor has not payed for
 
-   //kindly remove the statement below when you are done with the assignment
-    System.out.println("Hello world!");
+   //kindly remove the statement below when you are done with the assignment 
+     Scanner scanner = new Scanner(System.in);
+
+        double totalAmountPaid , daysPaidFor , unpaidAmount , daysNotPaidFor,amountPerDay, meantToPay,notCaptured, daysPaidForAmount;
+        System.out.println("ACCOUNT STATEMENT \n");
+
+        System.out.println("\n input amount per day \n" );
+        double ammountPerDay = scanner.nextDouble();
+
+
+        System.out.println("\n input days defaulted  \n");
+        double daysDefaulted = scanner.nextDouble();
+
+
+        System.out.println("\n DAYS \n");
+        double days = scanner.nextDouble();
+
+        System.out.println(" \n daysNotDefaulted \n ");
+        double daysNotDefaulted = scanner.nextDouble();
+
+
+        System.out.println("AMOUNT MEANT TO BE PAID: / TOTAL AMOUNT \n");
+        meantToPay = ammountPerDay * days ;
+        String totalFormatted = NumberFormat.getCurrencyInstance().format(meantToPay);
+        System.out.println("meantToPay :"+ totalFormatted);
+
+        System.out.println("DAYS PAID FOR  ");
+        daysPaidFor = days - daysDefaulted;
+        String accountFormatted = NumberFormat.getNumberInstance().format(daysPaidFor);
+        System.out.println("daysPaidFor :"+ accountFormatted );
+
+        System.out.println("DAYS PAID FOR AMOUNT \n");
+        daysPaidForAmount = ammountPerDay * daysNotDefaulted ;
+        String daysPaidForAmountFormatted = NumberFormat.getCurrencyInstance().format(daysPaidForAmount);
+        System.out.println("daysPaidForAmount :"+ daysPaidForAmountFormatted);
+
+        System.out.println("DAYS NOT CAPTURED ");
+        notCaptured = days - daysNotDefaulted;
```

Comment:
it is not days not captured but amount of the day not captured which should have been totalAmountPaid % ammountPerDay